k2/csrc/pytorch_context.cu

```diff
@@ -59,6 +60,26 @@ class PytorchCpuContext : public Context {
     return other.GetDeviceType() == kCpu;
   }
 
+  void CopyDataTo(size_t num_bytes, const void *src, ContextPtr dst_context,
+                  void *dst) override {
```

Comment:
This doesn't make sense to me, I thought we were taling about a member function of class Context.
How do we know where the source data is located?

Reply:
oh I see.. its in the class.

Reply:
mark as resolved.